packages/driver/src/cypress/error_messages.ts

```diff
@@ -983,6 +983,27 @@ export default {
 
         If so, increase \`redirectionLimit\` value in configuration.`
     },
+    switch_to_domain_load_timed_out ({ ms, configFile, crossOriginUrl, origins }) {
+      return stripIndent`\
+
+        Timed out after waiting \`${ms}ms\` for your remote page to load on origin(s):
+
+        ${origins.map((origin) => `\`${origin}\``).join('\n        ')}
+
+        Your page did not fire its \`load\` event within \`${ms}ms\`.
+
+        A cross origin request for \`${crossOriginUrl.href}\` was detected.
+
+        To handle cross origin requests, they must be immediately followed by a ${cmd('switchToDomain')} command:
+
+        \`cy.switchToDomain('${crossOriginUrl.origin}', () => { <cross origin commands go here> })\`
```

Comment:
```suggestion
        \`\`\`
        cy.switchToDomain('${crossOriginUrl.origin}', () => {
          <commands targeting ${crossOriginUrl.origin} go here>
        })
        \`\`\`
```

Unfortunately this doesn't display as multiple lines in the reporter, though it probably does in stdout at least. Need to add support for multi-line code blocks in reporter error messages at some point.

Reply:
updated